app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/HomeFragment.java

```diff
@@ -153,7 +153,7 @@ public void onResume() {
         super.onResume();
 
         int productCount = sp.getInt("productCount", 0);
-        apiClient.getTotalProductCount()
+        apiClient.getTotalProductCount(getString(R.string.app_name) + " " + Utils.getUserAgent("OTHERS"))
```

Comment:
We should use a common way to build the header.
For instance in openfoodfacts.github.scrachx.openfood.network.OpenFoodAPIService#getProductByBarcodeSingle we use only this info:
`client.getProductByBarcodeSingle(product.getCode(), fields, Utils.getUserAgent(Utils.HEADER_USER_AGENT_SEARCH))`

I suggest to remove `getString(R.string.app_name) + " "` and change Utils.getUserAgent to add a prefix if needed.

Reply:
Done